src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -41,7 +41,7 @@
     "d:transactions/measurements.lcp@millisecond": PREFIX + 103,
     "d:transactions/measurements.app_start_cold@millisecond": PREFIX + 104,
     "d:transactions/measurements.app_start_warm@millisecond": PREFIX + 105,
-    "d:transactions/measurements.cls@millisecond": PREFIX + 106,
+    "d:transactions/measurements.cls@none": PREFIX + 106,
```

Comment:
There are already multiple indexer entries for `d:transactions/measurements.cls@none` in production, so what will happen is:

* Filtering by this tag value will resolve to the new index, so pre-existing snuba data filed under this tag value will be hidden.
* Reverse-resolving pre-existing indexes (e.g. when processing a "group by" result) will still map to `d:transactions/measurements.cls@none`, so multiple groups with the same reverse-resolved tag value might appear.

As there will be at least one more full data wipe in production, it should still be OK to perform this change.